image/scanner/rhel/konflux.Dockerfile

```diff
@@ -10,6 +10,8 @@ ARG SCANNER_TAG
 RUN if [[ "$SCANNER_TAG" == "" ]]; then >&2 echo "error: required SCANNER_TAG arg is unset"; exit 6; fi
 ENV RELEASE_TAG="${SCANNER_TAG}"
 
+ENV GOEXPERIMENT=strictfipsruntime
+ENV GOTAGS=strictfipsruntime
```

Comment:
I forgot: do these both `GOEXPERIMENT=strictfipsruntime` env variable and `-tags=strictfipsruntime` argument need to be provided at the same time?

Reply:
I tried doing this in https://github.com/stackrox/stackrox/pull/12909, but only setting `GOEXPERIMENT` didn't move the needle (hence this commit: https://github.com/stackrox/stackrox/pull/12909/commits/0e094ea35b90da1473541c6ba205e61cfe779745)

Reply:
Weird. If `-tags=strictfipsruntime`/`GOTAGS=strictfipsruntime` has the effect, maybe we don't need `GOEXPERIMENT=strictfipsruntime`?

Reply:
What's the conclusion of this? Just keeping both?